libs/client.lua

```diff
@@ -1232,7 +1232,8 @@ packetListener = {
 	},
 	[44] = {
 		[1] = function(self, packet, connection, identifiers) -- Switch bulle identifiers
-			local bulleId = packet:read32()
+			local bulleCred1 = packet:read32()
+			local bulleCred2 = packet:read32()
```

Comment:
I don't really like the name of these variables...

Reply:
How would you name it then?